ethereum/smart-contract/test/gatewayToken.test.ts

```diff
@@ -346,45 +342,6 @@ describe('GatewayToken', async () => {
     });
   });
 
-  describe('network authorities', async () => {
-    it('Successfully add 1 new network authority to gatekeeper network', async () => {
-      await gatewayToken.connect(identityCom).addNetworkAuthority(networkAuthority2.address, gkn1);
-      expect(await gatewayToken.connect(identityCom).isNetworkAuthority(networkAuthority2.address, gkn1)).to.be.true;
-    });
-
-    it('Successfully add a gatekeeper after becoming network authority', async () => {
-      await gatewayToken.connect(networkAuthority2).addGatekeeper(gatekeeper.address, gkn1);
-    });
-
-    it('Expect revert when attempting to issue as a non-gatekeeper network authority', async () => {
-      await expect(
-        gatewayToken.connect(networkAuthority2).mint(alice.address, gkn1, 0, 0, NULL_CHARGE),
-      ).to.be.revertedWithCustomError(gatewayToken, 'Common__Unauthorized');
-    });
-
-    it("Try to remove non-existing network authorities, don't expect revert", async () => {
-      await gatewayToken
-        .connect(identityCom)
-        .removeNetworkAuthority('0x2F60d06Fa6795365B7b42B27Fa23e3e8c8b82f66', gkn1);
-    });
-
-    it('Remove a network authority', async () => {
-      await gatewayToken.connect(identityCom).removeNetworkAuthority(networkAuthority2.address, gkn1);
-    });
-
-    it('Expect revert on adding new network authority by Alice', async () => {
-      await expect(gatewayToken.connect(alice).addNetworkAuthority(bob.address, gkn1)).to.be.revertedWithCustomError(
-        gatewayToken,
-        'Common__Unauthorized',
-      );
-    });
-
-    it('Expect revert on removing existing network authority by Alice', async () => {
-      await expect(
-        gatewayToken.connect(alice).removeNetworkAuthority(identityCom.address, gkn1),
-      ).to.be.revertedWithCustomError(gatewayToken, 'Common__Unauthorized');
-    });
-  });
```

Comment:
Networks are now only managed by a single `primaryAuthority` so functionality around additional authorities and DAO management has been removed.